pandas/core/generic.py

```diff
@@ -7941,6 +7941,10 @@ def mask(self, cond, other=np.nan, inplace=False, axis=None, level=None,
         inplace = validate_bool_kwarg(inplace, 'inplace')
         cond = com._apply_if_callable(cond, self)
 
+        # see gh-21891
```

Comment:
I would do this in ``.where`` itself

Reply:
Do what exactly in `.where` ?

Reply:
oh i c we are using ``~`` here, ok then.